model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -22,6 +23,7 @@
 import org.neo4j.ogm.annotation.typeconversion.Convert;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
This is ignore any any additional properties that may come with publishedSampleJson. Used when the json is parsed to SampleMetadata in the sampleUpdateHandler

Reply:
What additional fields did you run into? Do you have a record of which ones the deserializer complained about?

Reply:
The alias fields